test/testcafe/spec/ChallengeFactorPassword_spec.js

```diff
@@ -53,6 +53,9 @@ test.requestHooks(authenticatorRequiredPasswordMock)(`challenge password authent
   await challengeFactorPageObject.forgotPasswordExists();
   await t.expect(challengeFactorPageObject.getForgotPasswordButtonText()).eql('Forgot password?');
 
+  await t.expect(await challengeFactorPageObject.signoutLinkExists()).ok();
```

Comment:
Signout should not exist on this page as well.

Reply:
see above. may need to revisit after talked to Vince.